docs/reference/ingest/processors/inference.asciidoc

```diff
@@ -0,0 +1,107 @@
+[role="xpack"]
+[testenv="basic"]
+[[inference-processor]]
+=== {infer-cap} Processor
+
+Uses a pre-trained {dfanalytics} model to infer against the data that is being 
+ingested in the pipeline.
+
+
+[[inference-options]]
+.{infer-cap} Options
+[options="header"]
+|======
+| Name               | Required  | Default                        | Description
+| `model_id   `      | yes       | -                              | (String) The ID of the model to load and infer against.
+| `target_field`     | no        | `ml.inference.<processor_tag>` | (String) The field name to which to write the results objects.
+| `field_mappings`   | yes       | -                              | (Object) Maps the document field names to the known field names of the model.
+| `inference_config` | yes       | -                              | (Object) Contains the inference type and its options. There are two types: `regression` and `classification`.
+include::common-options.asciidoc[]
+|======
+
+
+[source,js]
+--------------------------------------------------
+{
+  	"inference": {
+      	"model_id": "flight_delay_regression-1571767128603",
+      	"target_field": "FlightDelayMin_prediction_infer",
+      	"field_mappings": {},
+      	"inference_config": {"regression": {}},
+      	"model_info_field": "ml"
+    }
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+
+
+[discrete]
+[[inference-processor-regression-opt]]
+==== {regression-cap} configuration options
+
+`results_field`::
+(Optional, string)
+Specifies the field to which the inference prediction is written. Defaults to 
+`predicted_value`.
+
+
+[discrete]
+[[inference-processor-classification-opt]]
+==== {classification-cap} configuration options
+
+`results_field`::
+(Optional, string)
+Specifies the field to which the inference prediction is written. Defaults to 
```

Comment:
Related to earlier comment, if applicable, the wording in the enrich processor is nice:
```suggestion
The field that is added to incoming documents to contain the inference prediction. Defaults to 
```